zerver/lib/onboarding.py

```diff
@@ -131,6 +134,35 @@ def send_initial_direct_message(user: UserProfile) -> None:
         disable_external_notifications=True,
     )
 
+    custom_welcome_bot_message_string = ""
+    custom_welcome_bot_message_text = ""
+    # Send additional welcome bot message configured by administrators.
+    if custom_welcome_bot_message is not None:
+        custom_welcome_bot_message_text = custom_welcome_bot_message.strip()
+    elif (
+        user.realm.custom_welcome_bot_message_enabled
+        and user.realm.custom_welcome_bot_message is not None
+    ):
+        custom_welcome_bot_message_text = user.realm.custom_welcome_bot_message.strip()
+
+    if custom_welcome_bot_message_text:
+        custom_welcome_bot_message_string = (
+            _(
+                "The administrators for this organization would like to share the following information:\n"
+                "```quote\n{realm_custom_welcome_bot_message}\n```"
+            )
+            + "\n\n"
+        ).format(realm_custom_welcome_bot_message=custom_welcome_bot_message_text)
+
+        internal_send_private_message(
+            get_system_bot(settings.WELCOME_BOT, user.realm_id),
+            user,
```

Comment:
We can use `internal_prep_private_message()` twice, instead of using `internal_send_private_message()` twice. and then use `do_send_messages([default_message, custom_message])` -- This will save a lot of db queries.